<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>CHoldupsEditorClass</class>
  <widget class="QDialog" name="CHoldupsEditorClass">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1045</width>
        <height>600</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Holdups editor</string>
    </property>
    <property name="windowIcon">
      <iconset resource="../../DyssolGUI/Resources.qrc">
        <normaloff>:/Icons/WinIcon</normaloff>:/Icons/WinIcon</iconset>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4">
      <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
            <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
                <widget class="QLabel" name="label_4">
                  <property name="text">
                    <string>Units with holdups</string>
                  </property>
                  <property name="toolTip">
                    <string>List of units with holdups/inlet streams</string>
                  </property>
                  
                  <property name="whatsThis">
                    <string>List of units with holdups/inlet streams</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="CQtTable" name="modelsList">
                  <property name="editTriggers">
                    <set>QAbstractItemView::NoEditTriggers</set>
                  </property>
                  <property name="selectionMode">
                    <enum>QAbstractItemView::SingleSelection</enum>
                  </property>
                  <property name="selectionBehavior">
                    <enum>QAbstractItemView::SelectRows</enum>
                  </property>
                  <attribute name="horizontalHeaderVisible">
                    <bool>false</bool>
                  </attribute>
                  <attribute name="horizontalHeaderStretchLastSection">
                    <bool>true</bool>
                  </attribute>
                  <property name="toolTip">
                    <string>List of units with holdups/inlet streams</string>
                  </property>
                  
                  <property name="whatsThis">
                    <string>List of units with holdups/inlet streams</string>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
                <widget class="QLabel" name="label_6">
                  <property name="text">
                    <string>Holdups</string>
                  </property>
                  <property name="toolTip">
                    <string>List of holdups/inlet streams in selected unit</string>
                  </property>
                  
                  <property name="whatsThis">
                    <string>List of holdups/inlet streams in selected unit</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="CQtTable" name="holdupsList">
                  <property name="editTriggers">
                    <set>QAbstractItemView::NoEditTriggers</set>
                  </property>
                  <property name="selectionMode">
                    <enum>QAbstractItemView::SingleSelection</enum>
                  </property>
                  <property name="selectionBehavior">
                    <enum>QAbstractItemView::SelectRows</enum>
                  </property>
                  <attribute name="horizontalHeaderVisible">
                    <bool>false</bool>
                  </attribute>
                  <attribute name="horizontalHeaderStretchLastSection">
                    <bool>true</bool>
                  </attribute>
                  <property name="toolTip">
                    <string>List of holdups/inlet streams in selected unit</string>
                  </property>
                  
                  <property name="whatsThis">
                    <string>List of holdups/inlet streams in selected unit</string>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
        </layout>
      </item>
      <item>
        <widget class="CBasicStreamEditor" name="widgetHoldupsEditor" native="true" />
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>CQtTable</class>
      <extends>QTableWidget</extends>
      <header location="global">QtTable.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>CBasicStreamEditor</class>
      <extends>QWidget</extends>
      <header>BasicStreamEditor.h</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>modelsList</tabstop>
    <tabstop>holdupsList</tabstop>
  </tabstops>
  <resources>
    <include location="../../DyssolGUI/Resources.qrc" />
  </resources>
  <connections />
</ui>